src/components/form/Form.styeld.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
```

Comment:
Do you really need this `div` or could your transfer all these styles directly to your `form`?

Reply:
Did you think about my suggestion? 😉